Polymer/1802-Shadab/Polymer1.0_Demo/bower_components/iron-iconset-svg/demo/svg-sample-icons.html

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
You don't put all this stuff here. You only but bower file, the dependencies will be installed locally.